src/groundStation/commandParser.py

```diff
@@ -144,15 +163,29 @@ def __argCheck(self, args, inoutInfo, subservice=None):
 
         for i in range(0, len(args)):
             if inoutInfo['args'][i]:
-                nparr = np.array([args[i]], dtype=inoutInfo['args'][i])
+                if inoutInfo['args'][i] == 'var' and self._command['dport'] == 9 and outArgs[0] == 4:
```

Comment:
use port values set in system.py please
What does it mean if outArgs == 4?

Reply:
You mean make it work with 'Configuration' instead of 9 for example? I'm not sure if we have access to the dictionary in this function. `outArgs[0] == 4` means the support number is 4. I'm just being cautious here in case we start having other commands use 'var' too.

Reply:
You can do something like
```
self.vals['EPS']['PORT']
```

(not exactly, but this is how you should get the port number. But as per above, maybe we can just assume that if someone puts in a 'var' argument to another service then they know what they're going, and not bother restricting the port here.

Reply:
Oh, good to know.  Yeah, I'll just remove them for now.